kratos/geometries/line_3d_3.h

```diff
@@ -695,7 +469,7 @@ class Line3D3 : public Geometry<TPointType>
 
     SizeType FacesNumber() const override
     {
-        return 2;
+        return EdgesNumber();
```

Comment:
```suggestion
        return 0;
```
To be consistent with the linear line.

Reply:
In fact, I am doing this to be consistent with quadratic 2d line, should we change both? (in fact previous value was 2 already)

Reply:
I remember discussing about exactly this in the @KratosMultiphysics/technical-committee (long ago). Let me try to find the outcome of the discussion.

Reply:
Sorry, I didn't manage to find it...